test_schemas/en/test_interstitial_definition.json

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
Schema name shouldn't mention interstitial since it is not specific to interstitial.